### ISSUE TITLE
text for Scala 2.10 or 2.11?

### ISSUE THREAD

**Issue by xhudik** [scala/scala.github.com#422]
Hi,
I'm wondering whether ( https://github.com/scala/scala.github.com/blob/master/overviews/core/_posts/2012-09-20-futures.md )file is created for Scala 2.10 (I saw some label 2.10), or 2.11...
If it is for 2.10 -where it is possible to fid version for 2.11? 
(I suppose, page http://docs.scala-lang.org/overviews/core/futures.html should be updated with more recent version (for Scala 2.11)).

My original problem was that in the page `future` method is mentioned all the time (which is right for 2.10), however, there is `Future` method for version 2.11

e.g:
```
     val f: Future[List[String]] = future {
       session.getRecentPosts
     }
```

**Comment by xhudik** [comment]
I created a pull request: https://github.com/scala/scala.github.com/commit/0c59b2a583a28459ef8fec50239a1cac2de038c2 , however, i suppose there will be necessary to fix more problems than this one...

**Comment by SethTisue** [comment]
I think this got fixed in 25ee4040618bb5567da696aecb3cc478849d4b4a? I'm closing the issue but I looked at this too quickly and I'm wrong to close it, let me know.
